real_time/arduino_src/radio_buggy_mega/Makefile

```diff
@@ -381,6 +383,7 @@ program: $(TARGET).hex $(TARGET).eep
 # Program the device without an ISP by using the bootloader
 aprogram: $(TARGET).hex $(TARGET).eep
 	$(PYTHON) ./arduinoReset.py $(AVRDUDE_A_PORT)
+	$(PYTHON) filewrite.py

```

Comment:
We should move this to it's own rule and fix the dependency problem here. I don't think this will work because the program rule depends on (and will run after) the rule building the .hex file, which depends on compiling individual files.